apps/challenges/views.py

```diff
@@ -2705,3 +2706,29 @@ def manage_worker(request, challenge_pk, action):
             response_data = {"action": "Failure", "error": message}
 
     return Response(response_data, status=status.HTTP_200_OK)
+
+
+@api_view(["GET"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_presigned_url_for_annotations(request, challenge_phase_pk):
+    response_data = {
+            "error": "Sorry, you are not authorized for uploading an annotation file."
+        }
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+    challenge_phase = get_challenge_phase_model(challenge_phase_pk)
+
+    filename = ""
+    key = ""
```

Comment:
why is the key empty string for every request? is this intentional?

Reply:
Added the proper values.